js/views/modals/Settings/Advanced/Appearance.js

```diff
@@ -0,0 +1,37 @@
+import app from '../../../../app';
+import loadTemplate from '../../../../utils/loadTemplate';
+import baseVw from '../../../baseVw';
+
+export default class extends baseVw {
+  constructor( options = { } ) {
+    super( options );
+
+    this.options = options;
+    this.settings = app.settings.clone( );
```

Comment:
Actually, I don't think you would get the values in the way Josh described. Likely, you would just pass the model's toJSON() output as context to your template and then right in your template (`ob. macStyleWinControls`) you would have those values and could set the fields based off of them.

But, yeah, if for some reason in you're view you need to programmatically access the values, then it would be via app.localSettings.get().